torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -201,13 +258,19 @@ def __exit__(self, *args):
 
     def log_module_tracing_table_to_file(self):
         # ansi_escape is used to remove ANSI escape sequences in table used to make terminal output more readable
-
         ansi_escape = re.compile(r"\x1B\[[0-?]*[ -/]*[@-~]")
         table = ansi_escape.sub("", self.generate_module_tracing_table())
 
         with open("output.txt", "w") as log_file:
             log_file.write(table)
 
+    def log_operation_tracing_table_to_file(self):
+        ansi_escape = re.compile(r"\x1B\[[0-?]*[ -/]*[@-~]")
+        table = ansi_escape.sub("", self.generate_operation_tracing_table())
+
+        with open("output.txt", "w") as log_file:
```

Comment:
let's make the filename an argument. also in `log_module_tracing_table_to_file`.